src/EVPFFT/scripts/build-it.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash -e
+show_help() {
+    echo "Usage: source $(basename "$BASH_SOURCE") [OPTION]"
+    echo "Valid options:"
+    echo "  --machine=<darwin|chicoma|linux|mac>"
+    echo "  --heffte_build_type=<fftw|cufft|rocfft>"
+    echo "  --kokkos_build_type=<serial|openmp|pthreads|cuda|hip>"
+    echo "  --build_cores=<Integers with inclusive range of 1-32>. This argument is optional, default is set to 1"
+    echo "  --help: Display this help message"
+    return 1
+}
+
+# Initialize variables with default values
+machine=""
+heffte_build_type=""
+kokkos_build_type=""
+build_cores="1"
+fullmachine=""
```

Comment:
I dont think we need to define these fullXXX variables. we can just do --machine=$machine in the args for sourcing the other scripts. this keeps the scripts clean.